start_local.bat

```diff
@@ -100,11 +100,11 @@ echo Removing any existing containers with the same names
 docker rm -f charlie-mnemonic psdb charlie-mnemonic-python-env 2>nul
 
 echo Stopping any existing Docker containers
-docker-compose down 2>nul
+docker compose down 2>nul
 
 echo Starting Charlie Mnemonic using Docker Compose...
 echo First run takes a while
-docker-compose up --build -d
+dockercompose up --build -d
```

Comment:
This seems to be a typo

Reply:
thanks for cross checking this, thank you, I'll close this PR & open a different one to avoid populating the tree with small changes